ocs_ci/ocs/scale_lib.py

```diff
@@ -998,3 +1000,251 @@ def get_max_pvc_count():
+            Example, {'nodetype': 'app-pod'}
+
+    Returns:
+        pod_data (str): pod data with multiple PVC mount paths added
+
+    """
+
+    pods_list, temp_list = ([] for i in range(2))
+    for pvc_name in pvc_list:
+        temp_list.append(pvc_name)
+        if len(temp_list) == pvcs_per_pod:
+            pod_dict = constants.PERF_POD_YAML
+            pod_data = templating.load_yaml(pod_dict)
+            pod_name = helpers.create_unique_resource_name("scale", "pod")
+
+            # Update pod yaml with required params
+            pod_data["metadata"]["name"] = pod_name
+            pod_data["metadata"]["namespace"] = namespace
+            volume_list = pod_data.get("spec").get("volumes")
+            del volume_list[0]
+
+            if raw_block_pv:
+                device_list = pod_data.get("spec").get("containers")[0]["volumeDevices"]
+                del device_list[0]
+            else:
+                mount_list = pod_data.get("spec").get("containers")[0]["volumeMounts"]
+                del mount_list[0]
+
+            # Flag to add Liveness probe or DeploymentConfig and Liveness probe once
+            # to the pod_data yaml
+            flag = 1
+
+            for name in temp_list:
+                volume_name = f"pvc-{pvc_list.index(name)}"
+                volume_list.append(
+                    {
+                        "name": volume_name,
+                        "persistentVolumeClaim": {
+                            "claimName": f"{name}",
+                            "readOnly": False,
+                        },
+                    }
+                )
+                if raw_block_pv:
+                    device_path = f"{constants.RAW_BLOCK_DEVICE + name}"
+                    device_list.append({"name": volume_name, "devicePath": device_path})
+                else:
+                    mount_path = f"/mnt/{name}"
+                    mount_list.append({"name": volume_name, "mountPath": mount_path})
+
+                liveness_check_path = device_path if raw_block_pv else mount_path
+
+                if flag and deployment_config:
+                    # Update pod yaml with DeploymentConfig liveness probe and IO
+                    pod_data["kind"] = "DeploymentConfig"
+                    pod_data["apiVersion"] = "apps.openshift.io/v1"
+                    spec_containers = pod_data.get("spec")
+                    template_list = {
+                        "template": {"metadata": {"labels": {"name": pod_name}}}
+                    }
+                    pod_data["spec"] = template_list
+                    pod_data["spec"]["template"]["spec"] = spec_containers
+                    pod_data["spec"]["template"]["spec"]["restartPolicy"] = "Always"
+                    pod_data["spec"]["template"]["spec"]["containers"][0]["args"] = [
+                        "/bin/sh",
+                        "-c",
+                        f"fio --name=fio-rand-readwrite --filename={liveness_check_path}/abc "
```

Comment:
This is a good start. 
maybe in a subsequent PR, We should try to have a variety of IOs.